data/sql/updates/pending_db_world/rev_1689367513231192832.sql

```diff
@@ -0,0 +1,3 @@
+--
+UPDATE `creature_template` SET `mechanic_immune_mask` = `mechanic_immune_mask` | 8388608, `flags_extra` = 1073741824 WHERE `entry` in (19389, 21350);
```

Comment:
You can do the same as mechanic immune mask with flags_extra.

Reply:
flags_extra is empty though